salvo/install_deps.sh

```diff
@@ -14,10 +14,13 @@ fi
 
 /usr/bin/apt update
 /usr/bin/apt -y install \
+  containerd \
   docker.io \
   python3-pytest \
   python3-docker \
-  openjdk-11-jdk
+  openjdk-11-jdk \
+  python-apt \
+  lcov
```

Comment:
Keep in mind that the primary use of this script is to get the CI working.  It may be a better route to create a separate script for *user* dependencies vs *ci* dependencies. 

We could expand this script to be intelligent and detect the location where it's running and go from there.

Reply:
That is a better approach to take. Let me explore more on that.